aws/app/lambda/reliability/lib/markdown.js

```diff
@@ -14,10 +13,16 @@ module.exports = (formResponse, language) => {
     return { p: item };
   });
 
+  let formatedDate;
```

Comment:
Why do you have this try catch block ?

Reply:
In the event that where existing code ( unit tests etc...) depending on  this module  do not break due to  `submissionTimestamp`  type , nullability or format etc... Maybe a bit of extra cautions.

Reply:
The submissionTimestamp should be just a number. I think its safe to assume and rely on existing error handling.

Reply:
Otherwise you'd need this with every field